nan.h

```diff
@@ -281,20 +281,49 @@ NAN_INLINE uint32_t NanUInt32OptionValue(
 template<typename T>
 v8::Local<T> NanNew(v8::Handle<T>);
 
-template<typename T>
-NAN_INLINE v8::Local<T> _NanEnsureLocal(v8::Handle<T> val) {
-  return NanNew(val);
-}
+namespace NanIntern {
```

Comment:
Maybe name it nan::intern (or nan::internal or nan::imp or whatever) because it's going to show up in most editors when it's a top-level namespace.